tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -237,5 +238,15 @@
             </StackPanel>
         </ScrollViewer>
 
+        <muxc:InfoBar MaxWidth="480"

```

Comment:
Dev Home has a built-in [InfoBar](https://github.com/microsoft/devhome/blob/b3ca6571f5b8ba331dd6cc6c4858d51aef4b3999/src/Views/ShellPage.xaml#L107) and [InfoBarService](https://github.com/microsoft/devhome/blob/b3ca6571f5b8ba331dd6cc6c4858d51aef4b3999/src/Services/InfoBarService.cs#L10). I'm not sure why setupflow creates their own, but this one is available for you to use.

Reply:
Actually using Stacked Notifications, so can't use the service.